packages/launchpad/package.json

```diff
@@ -3,17 +3,26 @@
   "version": "0.0.0-development",
   "private": true,
   "scripts": {
-    "prebuild": "yarn codegen",
+    "prebuild": "rm -rf ./node_modules/.vite && yarn codegen",
     "build": "vite build",
     "types": "vue-tsc --noEmit",
     "build-prod": "cross-env NODE_ENV=production yarn build",
     "clean-deps": "rm -rf node_modules",
     "test": "yarn cypress:run && yarn types",
-    "cypress:launch": "cross-env TZ=America/New_York node ../../scripts/cypress open --project ${PWD}",
-    "cypress:open": "cross-env TZ=America/New_York node ../../scripts/cypress open-ct --project ${PWD}",
-    "cypress:run": "cross-env TZ=America/New_York node ../../scripts/cypress run-ct --project ${PWD}",
-    "dev": "NODE_ENV=development LAUNCHPAD=1 node ../../scripts/cypress open",
-    "postinstall": "yarn codegen && echo '@packages/launchpad needs: yarn build'",
+    "build-for-e2e": "kill -9 `lsof -t -i:52100` || echo ok && yarn test:e2e:serve & yarn build-prod",
```

Comment:
Can we update README to explain which of these command(s) are used for development and what they do?

Reply:
No, this will happen w Tim's PR.